src/services/target.services.js

```diff
@@ -0,0 +1,13 @@
+import client from '../helpers/client';
+
+const create = (radius, title, topic, lat, lon, user) => {
```

Comment:
do you need to send the user every time? You should have a function to get the current user, I think it would be easier, doesn't it?

Reply:
Do you mean to not set the user as a parameter, and get the user from the localStorage?

Reply:
exactly